tests/src/kernel/hipLanguageExtensions.cpp

```diff
@@ -53,7 +54,7 @@ __device__ __forceinline__ int sum1_forceinline(int a) { return a + 1; };
 
 __device__ __host__ float PlusOne(float x) { return x + 1.0; }
 
-__global__ void MyKernel(const hipLaunchParm lp, const float* a, const float* b, float* c,
```

Comment:
This change also was submitted and merged as part of the PR that replaces hipLaunchKernel with hipLaunchKernelGGL in all tests and samples.